python-qa-tools/load-test/multi_msg_load.py

```diff
@@ -0,0 +1,135 @@
+seq2, seq2err = command_processor(command)
+if len(seq2err):
+    sys.exit(seq2err)
+seq2 = json.loads(seq2)
+seq2no = int(seq2['sequence'])
+
+for i in range(0, int(NUM_TXS)):
+    unsignedTxto_command = f"{DAEMON} tx bank send {acc1} {acc2} 1000000{DENOM} --chain-id {CHAINID} --output json --generate-only --gas 500000"
+    unsignedTxto, unsignedTxtoerr = command_processor(unsignedTxto_command)
+    if len(unsignedTxtoerr):
+        sys.exit(unsignedTxtoerr) 
+
+    with open('unsignedto.json', 'w') as outfile:
+        json.dump(json.loads(unsignedTxto), outfile)
+    
+    unsignedTxfrom_command = f"{DAEMON} tx bank send {acc2} {acc1} 1000000{DENOM} --chain-id {CHAINID} --output json --generate-only --gas 500000"
+    unsignedTxfrom, unsignedTxfromerr = command_processor(unsignedTxfrom_command)
+    if len(unsignedTxfromerr):
+        sys.exit(unsignedTxfromerr)
+    with open('unsignedfrom.json', 'w') as outfile:
+        json.dump(json.loads(unsignedTxfrom), outfile)
+    
+    for j in range(0, int(num_msgs)):
+        write_json('unsignedto.json')
+        write_json('unsignedfrom.json')
+
+    ### seqto ###
+    seqto = seq1no + i
+    signTxto_command = f"{DAEMON} tx sign unsignedto.json --from {acc1} --chain-id {CHAINID} --keyring-backend test --home {DAEMON_HOME}-1 --node {RPC} --signature-only=false --sequence {seqto} --gas 500000"
+    signTxto, signTxtoerr = command_processor(signTxto_command)
+    
+    if len(signTxtoerr):
+        sys.exit(signTxtoerr)
+        
+    with open('signedto.json', 'w') as outfile:
+        json.dump(json.loads(signTxto), outfile)
+
+    broadcastto_command = f"{DAEMON} tx broadcast signedto.json --output json --chain-id {CHAINID} --gas 500000 --node {RPC} --broadcast-mode async"
+    broadcastto, broadcasttoerr = command_processor(broadcastto_command)
+    
+    if len(broadcasttoerr):
+        sys.exit(broadcasttoerr)
+    broadcastto = json.loads(broadcastto)
+    print(f"broadcasttoTxhash: {broadcastto['txhash']}")
+
+    ### seqfrom ###
+    seqfrom = seq2no + i
+    signTxfrom_command = f"{DAEMON} tx sign unsignedfrom.json --from {acc2} --chain-id {CHAINID} --keyring-backend test --home {DAEMON_HOME}-1 --node {RPC} --signature-only=false --sequence {seqfrom} --gas 500000"
+    signTxfrom, signTxfromerr = command_processor(signTxfrom_command)
+    
+    if len(signTxfromerr):
+        sys.exit(signTxfromerr)
+        
+    with open('signedfrom.json', 'w') as outfile:
+        json.dump(json.loads(signTxfrom), outfile)
+        
+    broadcastfrom_command = f"{DAEMON} tx broadcast signedfrom.json --output json --chain-id {CHAINID} --gas 500000 --node {RPC} --broadcast-mode async"
+    broadcastfrom, broadcastfromerr = command_processor(broadcastfrom_command)
+    
+    if len(broadcastfromerr):
+        sys.exit(broadcastfromerr)
+    broadcastfrom = json.loads(broadcastfrom)
+    print(f"broadcastfromTxhash: {broadcastfrom['txhash']}")
+
+print('##### Sleeping for 7s #####')
```

Comment:
add some documentation please

Reply:
Documentation as in inside the code or README Docs?
However, I have reduced the code lines for this function to a great extent. If you check the new commit, this code is not clumsy as it looks in this commit.

Reply:
In the code. Documentation is very much needed.